lib/ansible/plugins/loader.py

```diff
@@ -866,8 +855,6 @@ def get_with_context(self, name, *args, **kwargs):
             return get_with_context_result(None, plugin_load_context)
 
         fq_name = plugin_load_context.resolved_fqcn
-        if '.' not in fq_name:
```

Comment:
Since builtin plugin's resolved_fqcn always starts with `ansible.builtin.` now, all this did was break the fqcn for legacy - as it would end up being `.legacy` instead of `legacy`.

Reply:
This was fixed here https://github.com/ansible/ansible/commit/debf2be9131ccd7d5dd97798356b29fe66cb1312#diff-6790274bc6447a6667f1825052d04d79d60c098cc1e03a252372b56bbd5aa146R888, so I've removed it from this PR.